sql/dd/impl/types/object_table_impl.cc

```diff
@@ -58,8 +60,19 @@ Object_table_impl::Object_table_impl()
 }
 
 void Object_table_impl::set_encrypted() {
```

Comment:
I observed that this is used only for DD properties table and for the rest the constructor does this.

Reply:
And in the constructor the member variable is not set. This is a bug
I would suggest to call this set_encrypted() method from constructor